examples/text_embedder/android/app/src/main/java/com/google/mediapipe/examples/textembedder/MainActivity.kt

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.mediapipe.examples.textembedder
+
+import android.os.Bundle
+import android.view.View
+import android.widget.AdapterView
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.google.mediapipe.examples.textembedder.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity(), TextEmbedderHelper.EmbedderListener {
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var textEmbedderHelper: TextEmbedderHelper
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
+
+        textEmbedderHelper = TextEmbedderHelper(this, listener = this)
+
+        binding.btnCompare.setOnClickListener {
+            // Compare two texts here
+            val firstText =
+                if (binding.imgOne.text.isNullOrEmpty()) getString(R.string.default_compared_text) else binding.imgOne.text.toString()
+            val secondText =
+                if (binding.imgTwo.text.isNullOrEmpty()) getString(R.string.default_compared_text) else binding.imgTwo.text.toString()
```

Comment:
Can we create a separate/different default text for the second section? I used "I use MediaPipe to create amazing things with text, audio, and visual content" to get a 0.54 similarity. I think that gives us a nice example at the start with a difference rather than an immediate 1.0 value.